mkdocs.yml

```
@@ -28,6 +28,8 @@ nav:
       - 'Requesting your course space': 'requestCourse.md'
       - 'Creating content in the browser': 'getStarted.md'
       - 'Workshop': 'workshop/index.md'
+  - Student Guide:
+      - 'Accessibility': 'student-guide/accessibility.md'
   - User Guide:
```

If the above is the student guide this is probably the instructor guide...
```suggestion
  - Instructor Guide:
```